net-firewall/arno-iptables-firewall/arno-iptables-firewall-2.1.1.ebuild

```diff
@@ -0,0 +1,105 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit readme.gentoo-r1 systemd
+
+DESCRIPTION="Arno's iptables firewall script"
+HOMEPAGE="https://rocky.eld.leidenuniv.nl"
```

Comment:
Could you list the Github here as well for 2ndary HOMEPAGE?

Reply:
Thanks for the suggestion, @juippis, I agree it might make sense to point to the github repo as well.

I went ahead and included that in my [overlay](https://github.com/ferki/gentoo-overlay/commit/b55360d45e6422f8d59d83cee659b7b0e72487e1), so the next PR here will have it too. If you want me to send a followup PR before the next version bump with that as the sole change, let me know!